### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by krancour**

### CODE REVIEW

File: config_test.go [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+				_ restmachinery.APIClientOptions,
+				err error,
+			) {
+				require.Error(t, err)
+				require.Contains(t, err.Error(), "value not found for")
+				require.Contains(t, err.Error(), "API_TOKEN")
+			},
+		},
+		{
+			name: "SUCCESS not set",
```
Comment (vdice) [inline config_test.go:60]: nit: maybe just `success` 😄 ?

File: config.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	token, err := os.GetRequiredEnvVar("API_TOKEN")
+	if err != nil {
+		return address, token, opts, err
+	}
+	opts.AllowInsecureConnections, err =
+		os.GetBoolFromEnvVar("API_IGNORE_CERT_WARNINGS", false)
+	return address, token, opts, err
+}
+
+func tokenFilterConfig() (cloudevents.TokenFilterConfig, error) {
```
Comment (vdice) [inline config.go:33]: Should we add a unit test for this function?

File: internal/crypto/crypto_test.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+		name string
+		salt string
+	}{
+		{
+			name: "without salt",
+			salt: "",
+		},
+		{
+			name: "with salt",
+			salt: "n pepa",
```
Comment (vdice) [inline internal/crypto/crypto_test.go:20]: 👏

**Review by vdice — APPROVED** [review]
LGTM

File: config_test.go (hunk 1 above)
Comment (krancour) [reply config_test.go:60]: Not sure what happened there. Will fix.

File: internal/crypto/crypto_test.go (hunk 3 above)
Comment (krancour) [reply internal/crypto/crypto_test.go:20]: btw... This is a good candidate to get moved to brigade-foundations in a future PR.

File: config.go (hunk 2 above)
Comment (krancour) [reply config.go:33]: Yes... oversight that it isn't. Will fix.

**Pull request merged**